CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+
+## [1.5.3] - 2017-08-17
```

Comment:
generally it is best to leave the versioning to maintainers as there is no guarantee when it will be merged. I will let it slide this time as it is a small fix and I can get to it today.